components/eventing-controller/controllers/subscription/beb/reconciler_test.go

```diff
@@ -747,7 +747,8 @@ var _ = Describe("Subscription Reconciliation Tests", func() {
 			ensureAPIRuleStatusUpdatedWithStatusReady(ctx, &apiRuleCreated).Should(BeNil())
 
 			By("Setting a subscription not created condition")
-			subscriptionNotCreatedCondition := eventingv1alpha1.MakeCondition(eventingv1alpha1.ConditionSubscribed, eventingv1alpha1.ConditionReasonSubscriptionCreationFailed, v1.ConditionFalse, "")
+			message := "create subscription failed: 500; 500 Internal Server Error;{\"message\":\"sorry, but this mock does not let you create a BEB subscription\"}\n"
```

Comment:
(nit) This message looks more like log, would it be okay to show such formatted message in the conditions? Can we only extract the `error.message` part?

Reply:
I will have a look if that's possible. For now I print the entire error as string as you can see [here](https://github.com/raypinto/kyma/blob/0547523da9401f518dad3d06d9c6fc2319813461/components/eventing-controller/api/v1alpha1/condition.go#L229)